### PR TITLE
Feat/plugins settings

### PR DESCRIPTION
**Pull request by tupe12334**
Add plugins settings

### CODE REVIEW

File: plugins/db-mongo/src/index.ts [hunk 1]
```diff
@@ -101,7 +99,20 @@ class MongoPlugin implements AmplicationPlugin {
...
+    eventParams.envVariables = [
+      ...eventParams.envVariables,
+      ...[
+        { DB_USER: user },
+        { DB_PASSWORD: password },
+        { DB_PORT: port },
+        {
+          DB_URL: `mongodb://${user}:${password}@${host}:${port}/${dbName}?authSource=admin`,
+        },
+      ],
```
Comment (abrl91) [inline plugins/db-mongo/src/index.ts:114]: why did you change it? it is exactly as it was.

Comment (tupe12334) [reply plugins/db-mongo/src/index.ts:114]: Have a look at the values, it's not a string, its the values themselves.

Comment (abrl91) [reply plugins/db-mongo/src/index.ts:114]: but you can still use the constants as we did in other palaces to be consistent

Comment (tupe12334) [reply plugins/db-mongo/src/index.ts:114]: No we cant, because this values come from the plugin settings
